### PR TITLE
ROC-4052: Add `a11y` tests for claimant response feature

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-4052

### Change description
This PR adds `a11y` tests for claimant-response feature.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/test/a11y/a11y.ts [hunk 1]
```diff
@@ -92,7 +93,8 @@ const excludedPaths: DefendantResponsePaths[] = [
   DefendantResponsePaths.receiptReceiver,
   DefendantResponsePaths.legacyDashboardRedirect,
   OfferPaths.agreementReceiver,
-  DefendantFirstContactPaths.receiptReceiver
+  DefendantFirstContactPaths.receiptReceiver,
+  ClaimantResponsePaths.checkAndSendPage
```
Comment (geek-soft) [inline src/test/a11y/a11y.ts]: Why check-and-send is excluded?

Comment (sabahirfan) [reply src/test/a11y/a11y.ts]: not implemented yet.

Comment (geek-soft) [reply src/test/a11y/a11y.ts]: We just need to remember to revert it when it is implemented so we are checking it as a part of a11y tests.

Comment (timja-kainos) [reply src/test/a11y/a11y.ts]: can we not exclude it please? that's something easily forgotten

Comment (sabahirfan) [reply src/test/a11y/a11y.ts]: don't worry, it won't be forgotten, at this point it would be un necessary work-around to to include and make it pass.

**Pull request merged**